reference-architecture/azure-ansible/bastion.sh

```diff
@@ -190,94 +190,7 @@ cat > /home/${AUSERNAME}/setup-azure-master.yml <<EOF
     azure_conf_dir: /etc/azure
     azure_conf: "{{ azure_conf_dir }}/azure.conf"
     master_conf: /etc/origin/master/master-config.yaml
```

Comment:
master_conf variable is not longer used I think this can be safely removed too. Thoughts?